build.gradle.kts

```diff
@@ -23,14 +25,18 @@ object Versions {
     const val logstashEncoder = "7.0.1"
     const val mockk = "1.12.2"
     const val nimbusjosejwt = "9.18"
-    const val postgresEmbedded = "0.13.4"
+    const val postgresEmbedded = "1.0.0"
```

Comment:
Litt usikker på om vi kan gjøre dette. Det ble forsøkt å oppgradere denne tidligere, men da fikk man problemer med å kjøre tester lokalt (på Windows).

Reply:
Ja, best om dette reverseres.

Reply:
Jeg fikk det også til å kjøre i IntelliJ

Reply:
Jeg fikk det også til å kjøre i IntelliJ

Reply:
Funker dessverre *ikke* med 1.0.0 når jeg kjører ./gradlew clean build fra kommandolinja.

Reply:
Stemmer det @geir-waagboe sier og hos meg fungerer det ikke IntelliJ heller. Jeg får følgende feilmelding:
"Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration".

Når dere sier at det fungerer i Intellij hos dere på windows, er dere sikre på at dere har oppdatert depedency og at dere ikke har liggende v. 0.13.4 når dere kjøre test? Om jeg skal tippe har dere ikke oppdatert og hentet inn ny versjon.

I v. 1.0.0 så har de lagt til noe nye greier som krever et docker-env og ser ikke at dette fungerer automagisk på windows.